pkg/lint/lint.go

```diff
@@ -24,14 +24,14 @@ import (
 )
 
 // All runs all of the available linters on the given base directory.
-func All(basedir string, values map[string]interface{}, namespace string, strict bool) support.Linter {
+func All(basedir string, values map[string]interface{}, releaseName, namespace string, strict bool) support.Linter {
```

Comment:
Sorry, I missed this in the initial review. We don't want to change the signature of public API functions. As this will break existing SDK users.

Unfortunately I think it would be needed to add a new function. The easiest would be a `AllWithReleaseName`, which is (hopefully kinda obviously) a little ugly and doesn't help the next time extendability of the linter is needed. The better IMHO option would be to introduce a struct type, and use the https://golang.cafe/blog/golang-functional-options-pattern.html for the various linter config options.

Reply:
Yes, I agree that adding options here looks like a better option and it doesn't break the public API

Reply:
@gjenkins8 Hi, I apologize for the long delay, I was busy with work and completely forgot about this PR. I pulled up the main branch and did a little refactoring with the addition of options.  Could you review it again when you have some free time??